PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/Endpoint/memory_map.cpp

```diff
@@ -0,0 +1,164 @@
+#include "memory_map.h"
+
+MemoryMap::MemoryMap(uint64_t hostMemorySize, uint64_t deviceMemorySize)
+    : hostMemorySize_(hostMemorySize), deviceMemorySize_(deviceMemorySize)
+{
+    // Resize host memory to specified size
+    hostMemory_.resize(hostMemorySize_ / 4);
```

Comment:
What is the hostMemory, I mean what does it represent?

Reply:
check the latest commit